### PR TITLE
NYCT article basic positioning

### PR DESCRIPTION
**Pull request by SamanAtashi**
NYCT article is a clone of the actual [website](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0).

Features that have been used in this project include:
1. basic html/css structure
2. positioning element using tools such as (flex, position)
3. embedded a video

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @SamanAtashi :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there's still a little more to be done before this milestone can be approved.

## REQUIRED

* [ ] Elements on your page should be aligned with a combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)) and ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements#introduction)). You didn't make use of float and grid. Kindly see to that.
* [ ] Kindly get rid of the horizontal scroll bar at the bottom of your page. 

> ### "As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#) you have 2 code reviews left for this project."

## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Comment by SamanAtashi** [comment]
> @Elijah Ayandokun

Thank you for the feedback.
I added grid and float to the project but I can not find horizontal scrollbar.
Could you send me a screenshot or a guide regarding this issue?

File: style.css [hunk 1]
```diff
@@ -0,0 +1,1108 @@
...
+/*
+body {
+	background-color: white;
+	width: 100%;
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+} */
```
Comment (Berabjesus) [inline style.css]: - [ ] [optional] Please remove commented piece of codes. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: style.css [hunk 2]
```diff
@@ -0,0 +1,1108 @@
...
+  padding-top: 20px;
+  border-top: 1px solid black;
+}
+
+.icons button {
+  background-color: rgb(238, 238, 238);
+  border-radius: 50%;
+  border: 0;
+  cursor: pointer;
+} */
```
Comment (Berabjesus) [inline style.css]: - [ ] [optional] Please remove commented piece of codes. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: style.css [hunk 3]
```diff
@@ -0,0 +1,1108 @@
...
+/* .between .main-1 .box-1 {
+	border-top: 1px solid black;
+	height: 55px;
+	display: flex;
+	align-items: center;
+} */
+
+/* .between .main-1 .box-1 h5 {
+	border: 1px solid black;
+} */
```
Comment (Berabjesus) [inline style.css]: - [ ] [optional] Please remove commented piece of codes. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: style.css [hunk 4]
```diff
@@ -0,0 +1,1108 @@
...
+  height: 35px;
+  border: 1px solid black;
+  border-radius: 50%;
+  background-color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* .box-2 .small-box .row-1-small-pic button img { */
```
Comment (Berabjesus) [inline style.css]: - [ ] [optional] Please remove commented piece of codes. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: style.css [hunk 5]
```diff
@@ -0,0 +1,1108 @@
...
+
+.plus:hover {
+	background-color: black;
+	color: white;
+}
+
+.plus p:hover {
+	background-color: black;
+	color: white;
+}
```
Comment (Berabjesus) [inline style.css]: - [ ] [optional] Please remove commented piece of codes. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: style.css [hunk 6]
```diff
@@ -0,0 +1,1108 @@
...
+  font-family: 'Libre Franklin', sans-serif;
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 11px;
+} */
+
+/*! this is annoying line  */
+
+/* .black-underline a:hover {
+	text-decoration: underline;
```
Comment (Berabjesus) [inline style.css]: - [ ] [optional] Please remove commented piece of codes. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by Berabjesus — APPROVED** [review]
# **Status: PROJECT APPROVED** 
#### Excellent job on this project [@SamanAtashi](https://github.com/SamanAtashi). :fireworks: Keep it up.  :champagne: :champagne: **All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete. 👍** 
#### The [clone](https://samanatashi.github.io/NYCT_articles_2.0/) shows a good understanding and usage of Positioning and Floating elements. :100:
- The cloned page looks as it should and meets all the project specifications.
- The project has a descriptive README.md and is professionally done.
- Pull Request has a meaningful title and description.
- No linter errors
- Excellent semantic HTML 👏 
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
### **Before you merge to the main branch please have a look at the following OPTIONAL comments.**
- [ ] [optional] Please remove commented piece of codes. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [ ] [Optional] Avoid duplicate usage of rules, instead create one class for a rule and use it on the HTML. (For example, `display: flex` is used 36 times in the CSS, instead create a class for it, for example, `d-flex` and use that class in the HTML)

I wish you all the very best in your next tasks:1st_place_medal:
#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

**Comment by SamanAtashi** [comment]
Thanks a lot, man. that really helped. 🙏

**Comment by Berabjesus** [comment]
> Thanks a lot, man. that really helped. 🙏

You are welcome. 👍 
For future reference, don't delete the branches after working on them in case you have to make further changes. (personal opinion)

**Pull request merged**